app/services/openid_connect_redirector.rb

```diff
@@ -0,0 +1,75 @@
+class OpenidConnectRedirector
+  include ActionView::Helpers::TranslationHelper
+
+  def self.from_request_url(request_url)
```

Comment:
love this pattern of having a class method that instantiates a new object - so clean ✨